_news/105.md

```diff
@@ -32,7 +32,7 @@ Some highlights of the release are:
 * and much more ....
  You can find the Release Notes with more detailed info at
 
-  [ https://www.erlang.org/download/otp_src_19.0.readme](https://www.erlang.org/download/otp_src_19.0.readme)
```

Comment:
this is incorrect

Reply:
I see. This one is fixed here: https://github.com/erlang/erlang-org/pull/68/commits/7563aa70f277df9d86a7604f1c9ca2e23111f395